### ISSUE TITLE
Display list of Neatline exhibits on items/show that refer to a given item

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/Neatline#266]
Any time an Omeka item is used in a Neatline exhibit, display a link to that exhibit on the items/show view. Would help make connection between Neatline and Omeka archive more prominent and aid in discovering exhibits through the archive.

**Comment by davidmcclure** [comment]
Implemented with 501693ac2ea5f3c9aa6a121fd21fbcaeaaad6c91.

**Comment by jeremyboggs** [comment]
This doesn't appear to do what I describe. I suggested displaying a list of Neatline xhibits on the omeka item/show template, not the template in individual Neatline exhibits.

**Comment by davidmcclure** [comment]
Ah, sorry, I misread the ticket. This is a good feature - to do it well, we'd also need to implement durable-linking to individual records inside of exhibits (eg, `/show/exhibit-slug/40` would load the exhibit and auto-focus on record 40). This should be in there anyway, and it would just be a matter of adding a router to the public map module.

**Comment by jeremyboggs** [comment]
Yeah, I thought this was already in Neatline. Would be very nice to have if it isn't, but I'd also say for now we could simply display a link to the home page of a Neatline Exhibit if it contains that Omeka item.

**Comment by omegader** [comment]
2nd this.

The "View the item in Omeka" link in Neatline should work both ways. In other words, a "View the item in Neatline" link at the end of the Omeka item page. Ideally, this link would snap to the individual record and could be styled or hidden like any other Omeka element. But I would settle for less.
